lib/pg_eventstore/callbacks.rb

```diff
@@ -0,0 +1,122 @@
+# frozen_string_literal: true
+
+module PgEventstore
+  # Allows you to define before, around and after callbacks withing the certain action. It is especially useful during
+  # asynchronous programming when you need to be able to react on asynchronous actions from the outside.
+  # Example:
```

Comment:
Since subscriptions implementation is asynchronous - we need to react of certain events outside the subscription's runner(thread) life cycle. I simply need the ability to persist the proc and run it later, at the certain point